src/VectorizeLoops.cpp

```diff
@@ -1019,10 +1127,9 @@ class VectorizeLoops : public IRMutator {
                            << "constant extent > 1\n";
             }
 
-            // Replace the var with a ramp within the body
-            Expr for_var = Variable::make(Int(32), for_loop->name);
-            Expr replacement = Ramp::make(for_loop->min, 1, extent->value);
-            stmt = VectorSubs(for_loop->name, replacement, in_hexagon, target).mutate(for_loop->body);
+            vectorized_vars.push_back({for_loop->name, for_loop->min, (int)extent->value});
+            stmt = VectorSubs(vectorized_vars, in_hexagon, target).mutate(for_loop->body);
+            vectorized_vars.clear();
```

Comment:
Shouldn't this be a `pop_back()` ?

Reply:
This specific vector never had more than var in it, so clear() worked, however, vector actually is not needed here, so I just removed it.